zigate/core.py

```diff
@@ -57,7 +57,7 @@ def __getattr__(self, *args, **kwargs):
 BIND_REPORT = True  # automatically bind and report state for light
 SLEEP_INTERVAL = 0.1
 ACTIONS = {}
-WAIT_TIMEOUT = 3
+WAIT_TIMEOUT = 10
```

Comment:
10 sec is too much IMHO, 5sec could be a improvement

Reply:
Done.